_faq/installation_and_deployment/separate-project.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Why run Forseti in its own separate project?
+order: 4
+---
+{::options auto_ids="false" /}
+
+Setting Forseti up in a separate project ensures that Forseti has the necessary 
+quota for API calls and managing service accounts and roles. This also 
+restricts who has access to your Forseti-related resources.
```

Comment:
Is there a relevant section where we can point the user to in the best practices document that explains our rationale for this?